Session.js

```diff
@@ -20,12 +25,16 @@ export default class Session {
    * `authentication` is passed or if the refresh causes the session data
    * to change, then a `change` event is emitted.
    *
-   * @param {object} authentication - Authentication information to pass to
-   *   change listeners.
+   * @param {object} options - Options to use.
+   * @param {object} options.authentication - Authentication information to
+   *   pass to change listeners.
    */
   async refresh({authentication} = {}) {
     const oldData = this.data;
     const newData = await this._service.get();
+    if(typeof newData.originalMaxAge === 'number') {
```

Comment:
I'm assuming this will change to use `newData.ttl` instead.

Reply:
we do need a way to set the timeout using expires in case someone does want to use a timestamp over tll. I went ahead and changed to `ttl` and made `_setTimeout` private. https://github.com/digitalbazaar/bedrock-web-session/pull/5/commits/3ba295a12e0a1309909774da1a5da905d965e1ef